ajax/libs/tent-css/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "tent-css",
+  "filename": "tent.min.css",
+  "description": "A CSS survival kit.",
+  "author": "Aaron Mazade <ulinaaron@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sitetent/tentcss.git"
+  },
+  "homepage": "https://css.sitetent.com",
+  "keywords": [
+    "css",
+    "flexbox",
+    "html",
+    "html5",
+    "scss",
+    "sass",
+    "responsive",
+    "framework"
+  ],
+  "npmName": "tent-css",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(06_components)/**/*",
```

Comment:
Let's use one simple rule `!(06_components){,**/*}` :smile: 
It means we want to fetch all files under `dist` except `06_components` itself and sub-folders and files inside.